iml-manager-cli/src/stratagem.rs

```diff
@@ -323,21 +332,68 @@ pub async fn stratagem_cli(command: StratagemCommand) -> Result<(), ImlManagerCl
 
             wait_for_cmd_display(command).await?;
         }
-        StratagemCommand::Filesync(data) => {
-            let query = stratagem_queries::filesync::build(
-                &data.filesystem,
-                data.remote,
-                data.expression,
-                "push",
-            );
-
-            let resp: iml_graphql_queries::Response<stratagem_queries::filesync::Resp> =
-                graphql(query).await?;
+        StratagemCommand::Filesync(data) => match data.expression {
+            Some(ref _x) => {
```

Comment:
At this point, you actually have the expression, no the line below is not necessary.